sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/ServiceItemLease.java

```diff
@@ -203,7 +200,7 @@ public static ServiceItemLease fromDocument(CosmosItemProperties document) {
 
         String leaseTimestamp = ModelBridgeInternal.getStringFromJsonSerializable(document,PROPERTY_NAME_TIMESTAMP);
         if (leaseTimestamp != null) {
-            return lease.withTimestamp(ZonedDateTime.parse(leaseTimestamp));
+            return lease.withTimestamp(ZonedDateTime.parse(leaseTimestamp).toInstant());
```

Comment:
why are we doing translation from ZoneDateTime to Instant? can't we rely on Instant everywhere?

@simplynaveen20 @milismsft